### DISCUSSION TITLE
Assistant with husk related talents instead of clown.

### DISCUSSION THREAD

**Discussion by Hazmat616** [FakeFishGames/Barotrauma#12227] [Ideas and suggestions]
Basically assistants currently have clown related talents but nothing related to husk infections that players or npcs of the church of the husk would use. So even if there was simply a duplicate job with identical talents that aren’t clown related and having those replaced with church or husk oriented one’s would be nice. Either with a different name or color so you know what you are getting in the hiring/selection screen.

**Comment by G-M-twostay** [comment]
clowns are currently pretty lacking in comparison to the husks quest line, so assistants having clown talents is fair as it adds more clown contents.

**Reply by Hazmat616** [reply]
They could honestly use an update that pits them against each other more as well as overlap what outposts they can be at so they occasionally fight.
